rfc/updates.md

```diff
@@ -194,6 +194,51 @@ Note that:
   Images are specified by the CLI upon loading the Helm chart, by inspeciting `constellation-conf.yaml`.
   Deployment variations could be introduced into the Helm charts if they become necessary in the future.
 
+### CLI commands
+
+`constellation upgrade` has 2 sub commands:
+
+* `constellation upgrade check`
+* `constellation upgrade apply`
+
+When `constellation upgrade check` is called it checks if the current CLI embeds helm charts and kubernetes components which are newer than the ones defines in the `constellation-config.json`. If this is the case, it prints a detailed list of all the components that will be updated. Moreover, it checks via the update API (see: rfc/update-api.json) for new image patch versions as they are always compatible with inside a minor version.
+
+Lastly, it uses the update API (see: rfc/update-api.json) if there is a newer CLI version available. If this is the case, it will print the latest version and omit ony of the points above since. If this version is more than +1 minor version apart it will also show the latest CLI of the next minor version, and suggest a way to download it. Note that if there are still e.g. microservice updates needed with the current CLI, we need to promt the user to first install those before continuing with the next minor release.
```

Comment:
```suggestion
Lastly, the CLI checks if a newer CLI version is available via the update API (see: rfc/update-api.json). If this is the case, it will print the latest CLI version instead of the output described above. 
If the difference between `current_version` and `latest_version` is greater than one it will also show the latest CLI of the next minor version, and suggest a way to download it. 
An example:

current CLI version: `2.3.2`  
available CLI version with minor version `2.4`: `2.4.1`, `2.4.2`, `2.4.3`  
latest CLI version: `2.5.0`  
`constellation upgrade check` will show `2.5.0` as latest, and suggest that the next step in the upgrade process is `2.4.3`.  

The need to download CLI version `2.4.3` first comes from the fact that each minor version of the CLI is only able to upgrade from `current_version-1` to `current_version`.
If there are still microservice updates needed with the current CLI, we need to prompt the user to first install those before continuing with the next minor release.
```